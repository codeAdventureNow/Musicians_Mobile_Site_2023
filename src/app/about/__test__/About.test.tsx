import renderer from 'react-test-renderer';
import About from '../page';

describe('About', () => {
  it('should render the "About" page', () => {
    const aboutPage = renderer.create(<About />).toJSON();
    expect(aboutPage).toMatchSnapshot();
  });
});
