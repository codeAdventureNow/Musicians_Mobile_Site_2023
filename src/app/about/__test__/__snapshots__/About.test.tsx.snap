// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render the "About" page 1`] = `
<div
  className="homepage"
>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h2
        className="section_heading"
      >
        About 
        <span
          className="hero_heading_highlight"
        >
          Us
        </span>
         
      </h2>
      <div
        className="flex"
      >
        <div
          className="content"
        >
          <p
            className="paragraph"
          >
            “We aim to save your family time for what matters most by bringing high-quality music instructors to your home. We specialize in tailoring our lessons to your child’s interests to help them connect to the instrument and establish a life-long love of music."
          </p>
          <p>
            "On this path, we help to developing musical proficiency through a repertoire of songs, while honing their collaborative skills through regular live performance opportunities. We are honored to help every one of our students gain confidence within themselves through the joy of music.”
          </p>
          <p
            className="signature"
          >
            <span>
              Nate Mueller
            </span>
             Founder
          </p>
          <a
            className="cta_button"
            data-testid="schedule-button"
            href="/contact"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
            rel="noreferrer"
          >
            Schedule A Free Lesson!
          </a>
        </div>
        <div
          className="intro_image_container"
        >
          <img
            alt="Nate Mueller"
            className="image margin_top"
            data-nimg="1"
            decoding="async"
            height={200}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimages%2FNate_Pic.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimages%2FNate_Pic.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FNate_Pic.jpg&w=640&q=75 2x"
            style={
              {
                "color": "transparent",
                "height": "auto",
                "maxWidth": "100%",
              }
            }
            width={200}
          />
        </div>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h2
        className="section_heading"
      >
        Musicians
        <span
          className="hero_heading_highlight"
        >
           Mobile
        </span>
         
      </h2>
      <div
        className="flex_reverse"
      >
        <div
          className="content"
        >
          <p
            className="paragraph"
          >
            Musician’s Mobile provides professional music lessons in the comfort of your home. Our music and voice instructors teach a variety of instruments in many different styles.
             
            <span>
               
              We pride ourselves on teaching students of many ages how to play their favorite songs while giving them the essential skills necessary to become a well-rounded musician.
            </span>
          </p>
          <p>
            Musicians Mobile was founded by Nate Mueller in 2009 with the goal of providing enjoyable home music lessons. The company offers a collection of music instructors who share a similar passion for teaching. We work together to create a positive musical environment that helps students grow through lessons, recitals and musical workshops.
          </p>
        </div>
        <div
          className="intro_image_container"
        >
          <img
            alt="Brandon teaching"
            className="image margin_top"
            data-nimg="1"
            decoding="async"
            height={225}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimages%2FBrandon_InHome_Piano.png&w=640&q=75"
            srcSet="/_next/image?url=%2Fimages%2FBrandon_InHome_Piano.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FBrandon_InHome_Piano.png&w=640&q=75 2x"
            style={
              {
                "color": "transparent",
                "height": "auto",
                "maxWidth": "100%",
              }
            }
            width={255}
          />
        </div>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h2
        className="section_heading"
      >
        Student
        <span
          className="hero_heading_highlight"
        >
           Success
        </span>
         
      </h2>
      <div
        className="column"
      >
        <div
          className="intro_image_container"
        >
          <img
            alt="student performing"
            className="image"
            data-nimg="1"
            decoding="async"
            height={225}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimages%2FArch_Piano.jpg&w=640&q=75"
            srcSet="/_next/image?url=%2Fimages%2FArch_Piano.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FArch_Piano.jpg&w=640&q=75 2x"
            style={
              {
                "color": "transparent",
                "height": "auto",
                "maxWidth": "100%",
              }
            }
            width={200}
          />
        </div>
        <div
          className="content margin_top"
        >
          <p
            className="paragraph italic"
          >
            “I must thank you for the opportunity to perform on Saturday.
             
            <span
              className="bold"
            >
              It felt nice to see smiles on the wonderful audience’s faces. Archis and Ronan worked hard and loved every minute.
            </span>
             
            Please do let us know of future opportunities like this. I would love for them to participate. Archis is really loving the singing part of it and Ronan being so little is enjoying sitting at the piano for hours and practicing. Thanks again! You have a great team. We absolutely love Liana and Jared is really great .”
          </p>
          <p
            className="signature"
          >
            <span>
              Guarangi C. 
            </span>
             Los Gatos, CA
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
